sdk/storage/Azure.Storage.Common/tests/StreamPartitionerTests.cs

```diff
@@ -223,6 +223,13 @@ public override long Position
 
         class MockNonSeekableStream : Stream
         {
+            static int seed = Environment.TickCount;
+
+            static readonly ThreadLocal<Random> random =
+                new ThreadLocal<Random>(() => new Random(Interlocked.Increment(ref seed)));
+
+            public static Random Random => random.Value;
+
```

Comment:
It's the only place that particular version of `Random` is being used.  Every other random value has been made a lot less random as part of the recorded tests.  Removing this from the shared project makes it less likely anyone will use it instead of `this.Recording.Random`.